src/main/cpp/t04_neighbors.cpp

```diff
@@ -21,5 +21,18 @@
 using namespace std;
 
 int t04_neighbors() {
-
+    int N ;
+    cin >> N ;
+    int v[100] ;
+    cin >> v[0] ;
+    for ( int i = 1 ; i < N ; ++i )
+    {
+        cin >> v[i] ;
+        if (( v[i - 1] < 0 && v[i - 1] < 0 ) || ( v[i - 1] > 0  && v[i] > 0 ))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Same expression on both sides of '&&'.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4732197)